src/OpenTelemetry/Metrics/Processors/MetricConsoleExporter.cs

```diff
@@ -24,6 +24,11 @@ public class MetricConsoleExporter : MetricProcessor
     {
         private string name;
 
+        public MetricConsoleExporter()
```

Comment:
It is a bit confusing to have the exporter class derived from processor class.
Not introduced by this PR so no need to block on it.

Reply:
yea. This is something that would be changed.
(Also the ConsoleExporter is *not* coming from the Exporter.Console package.. Will be fixing all of them in subsequent PRs)